src/quo2/components/settings/category_list/component_spec.cljs

```diff
@@ -0,0 +1,35 @@
+(ns quo2.components.settings.category-list.component-spec
+  (:require [quo2.components.settings.category-list.view :as category-list]
+            [test-helpers.component :as h]))
+
+
+(h/describe "Category List component"
+            (h/test "renders category list component"
+                    (h/render [category-list/category-list
+                               {:title "Test Title"
+                                :on-press #(println "Pressed")
```

Comment:
you should not need these empty props for the test here, please just include only the essential props for the test 👍